src/test/resources/Features/OCD-2114-SLIFilterCheckedByDefault.feature

```diff
@@ -6,14 +6,14 @@ Scenario: Verify SLI Compliance filter option under ACB filters is checked by de
 	Then I see that "SLI Compliance" checkbox is checked
 
 Scenario Outline: Verify SLI Compliance filter option is checked by default on collections pages
-	Given I am on "<PageTitle>" collections page: "<PageName>"
+	Given I am on collections page: "<PageName>" on "DEV"
```

Comment:
I don't like that these steps are now specific to DEV. If I wanted to run this test locally I'd have to change the feature file, and that's not okay. That said, I'm not going to hold up the approval any longer on this because even if it's not 100% correct it's close enough to make progress on